Fix PyMySQL workarounds

# AsIs
As commented in https://github.com/hwakabh/bennu-official/pull/37#issuecomment-1864506648, we have the workaround to allow Django to use PyMySQL instead of default mysqlclient, with adding the following codes to `manage.py`.

```python
import pymysql
pymysql.install_as_MySQLdb()
```

This workarounds would be only applicable for local development, meaning `runserver` and `migration`, so that this triggers application startup crashes in production environment with gunicorn.

```bash
% gunicorn --bind 0.0.0.0:8000 bennu_official.wsgi
[2023-12-21 01:27:10 +0900] [52956] [INFO] Starting gunicorn 21.2.0
[2023-12-21 01:27:10 +0900] [52956] [INFO] Listening at: http://0.0.0.0:8000 (52956)
[2023-12-21 01:27:10 +0900] [52956] [INFO] Using worker: sync
[2023-12-21 01:27:10 +0900] [52957] [INFO] Booting worker with pid: 52957
[2023-12-21 01:27:11 +0900] [52957] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 15, in <module>
    import MySQLdb as Database
ModuleNotFoundError: No module named 'MySQLdb'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
                ^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
                    ^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
           ^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/.pyenv/versions/3.11.5/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/hwakabayashi/git/bennu-official/bennu_official/wsgi.py", line 7, in <module>
    application = get_wsgi_application()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/.pyenv/versions/3.11.5/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/contrib/auth/models.py", line 3, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/contrib/auth/base_user.py", line 58, in <module>
    class AbstractBaseUser(models.Model):
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 143, in __new__
    new_class.add_to_class("_meta", Options(meta, app_label))
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/db/models/base.py", line 371, in add_to_class
    value.contribute_to_class(cls, name)
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/db/models/options.py", line 243, in contribute_to_class
    self.db_table, connection.ops.max_name_length()
                   ^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/utils/connection.py", line 15, in __getattr__
    return getattr(self._connections[self._alias], item)
                   ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/utils/connection.py", line 62, in __getitem__
    conn = self.create_connection(alias)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/db/utils.py", line 193, in create_connection
    backend = load_backend(db["ENGINE"])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/db/utils.py", line 113, in load_backend
    return import_module("%s.base" % backend_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/.pyenv/versions/3.11.5/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/hwakabayashi/git/bennu-official/.venv/lib/python3.11/site-packages/django/db/backends/mysql/base.py", line 17, in <module>
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module.
Did you install mysqlclient?
[2023-12-21 01:27:11 +0900] [52957] [INFO] Worker exiting (pid: 52957)
[2023-12-21 01:27:11 +0900] [52956] [ERROR] Worker (pid:52957) exited with code 3
[2023-12-21 01:27:11 +0900] [52956] [ERROR] Shutting down: Master
[2023-12-21 01:27:11 +0900] [52956] [ERROR] Reason: Worker failed to boot.
```

# ToDo
Move the code block from `manage.py` to `settings.py`, which is loaded in production environment.
Ref: https://github.com/hwakabh/bennu-official/pull/37/files#diff-57f701b24994e3ca4194f54015c59a3bc690a0f9a9c23617980b952c979fe8b7R5-R6

# (Optional) Acceptance criteria
App would run without any error by `gunicorn` commands.

--- a/bennu_official/settings.py
+++ b/bennu_official/settings.py
@@ -60,6 +60,9 @@ WSGI_APPLICATION = 'bennu_official.wsgi.application'
 
 
 # Database
+import pymysql
+pymysql.install_as_MySQLdb()
+
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',

--- a/manage.py
+++ b/manage.py
@@ -2,9 +2,6 @@
 import os
 import sys
 
-import pymysql
-pymysql.install_as_MySQLdb()
-
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "bennu_official.settings")

